spec/services/supplier_locations/importer_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SupplierLocations::Importer do
+  subject(:call_importer) { importer.call }
+
+  let(:importer) { described_class.new(filename) }
```

Comment:
we use importer only in the subject,
maybe we can simplify it with just single line: `subject(:call_importer) { described_class.new(filename).call }`